examples/peripherals/lcd/i80_controller/main/i80_controller_example_main.c

```diff
@@ -55,15 +55,22 @@ static const char *TAG = "example";
 #define EXAMPLE_PIN_NUM_BK_LIGHT       1
 
 // The pixel number in horizontal and vertical
-#define EXAMPLE_LCD_H_RES              240
-#define EXAMPLE_LCD_V_RES              280
 // Bit number used to represent command and parameter
 #if CONFIG_EXAMPLE_LCD_I80_CONTROLLER_ST7789
+#define EXAMPLE_LCD_H_RES              240
+#define EXAMPLE_LCD_V_RES              280
 #define EXAMPLE_LCD_CMD_BITS           8
 #define EXAMPLE_LCD_PARAM_BITS         8
 #elif CONFIG_EXAMPLE_LCD_I80_CONTROLLER_NT35510
+#define EXAMPLE_LCD_H_RES              240
+#define EXAMPLE_LCD_V_RES              280
 #define EXAMPLE_LCD_CMD_BITS           16
 #define EXAMPLE_LCD_PARAM_BITS         16
+#elif CONFIG_EXAMPLE_LCD_I80_CONTROLLER_ILI9341
+#define EXAMPLE_LCD_H_RES              320
+#define EXAMPLE_LCD_V_RES              240
```

Comment:
Even for the same LCD controller, we can't guarantee the resolution will be the same for all LCMs, maybe your ILI9341's resolution is 320*240, but mine is 280*240. It's impossible to make the example work out of the box for all LCD break-out boards, without changing some of these default values :thinking: That's why I didn't make the resolution configuration strong associated with the LCD model.